### PR TITLE
Let iOS flutter run auto-sign default to first profile

### PR DESCRIPTION
**Pull request by xster**
#9589

### CODE REVIEW

File: packages/flutter_tools/lib/src/base/terminal.dart [hunk 1]
```diff
@@ -90,16 +91,29 @@ class AnsiTerminal {
...
+    int defaultChoiceIndex,
     bool displayAcceptedCharacters: true,
     Duration timeout,
   }) async {
     assert(acceptedCharacters != null);
     assert(acceptedCharacters.isNotEmpty);
+    List<String> charactersToDisplay = acceptedCharacters;
+    if (defaultChoiceIndex != null) {
+      charactersToDisplay = new List<String>.from(charactersToDisplay);
+      charactersToDisplay[defaultChoiceIndex] = bolden(charactersToDisplay[defaultChoiceIndex]);
```
Comment (xster) [inline packages/flutter_tools/lib/src/base/terminal.dart]: this syntax annoys me. Wish there's a way to do charactersToDisplay[defaultChoiceIndex].=do(bolden) or something

**Comment by Hixie** [comment]
cc @tvolkert

File: packages/flutter_tools/lib/src/base/terminal.dart [hunk 2]
```diff
@@ -90,16 +91,29 @@ class AnsiTerminal {
...
+      assert(defaultChoiceIndex >= 0 && defaultChoiceIndex < acceptedCharacters.length);
```
Comment (tvolkert) [inline packages/flutter_tools/lib/src/base/terminal.dart]: Move the assert up to the start of the `if` block

**Review by tvolkert — APPROVED** [review]
LGTM

**Review by cbracken — APPROVED** [review]
lgtm with one tiny nit :)

File: packages/flutter_tools/lib/src/base/terminal.dart (hunk 2 above)
Comment (xster) [inline packages/flutter_tools/lib/src/base/terminal.dart]: done

File: packages/flutter_tools/lib/src/base/terminal.dart [hunk 3]
```diff
@@ -90,16 +91,29 @@ class AnsiTerminal {
...
+  /// `prompt` is the text displayed prior to waiting for user input each time.
+  /// `defaultChoiceIndex`, if given, will be the character in `acceptedCharacters`
+  ///     in the index given if the user presses enter without any key input.
+  /// `displayAcceptedCharacters` prints also the accepted keys next to the `prompt` if true.
+  ///
   /// Throws a [TimeoutException] if a `timeout` is provided and its duration
   /// expired without user input. Duration resets per key press.
   Future<String> promptForCharInput(
-    List<String> acceptedCharacters, {
+    @required List<String> acceptedCharacters, {
```
Comment (xster) [inline packages/flutter_tools/lib/src/base/terminal.dart]: ah oops

**Pull request merged**